src/services/cooperation-service.ts

```diff
@@ -58,6 +59,18 @@ export const cooperationService = {
         parameters: { id }
       })
     })
+  },
+  updateResourceCompletionStatus: async (
+    data: UpdateResourceCompletionStatusParams
+  ) => {
+    return baseService.request<void>({
+      data,
```

Comment:
Don't send `cooperationId` and `resourceId` in the body of the request, you already sent it with URL parameters
```suggestion
      data: { completionStatus },
```

Reply:
fixed